jenkins/ci.suse.de/openstack-mkcloud.yaml

```diff
@@ -283,10 +283,25 @@
     - shell: |
         set -x
         shopt -s extglob
+
+        echo $WORKSPACE
+        pwd
+
         # emptying the workspace
         mkdir -p empty
         rsync -r --delete empty/ ./
 
+        cat <<'EOF' > log-parser-plugin-rules.txt
+        start /^(===+> MKCLOUD STEP START: |MKCLOUD step: )/
```

Comment:
I thought that you only wanted to match this: `MKCLOUD step:`
With the marker in as well there are too many sections now, or what am I missing?